packages/js/src/Modules/Verto/webrtc/helpers.ts

```diff
@@ -368,6 +368,210 @@ const sdpBitrateHack = (
   return lines.join(endOfLine);
 };
 
+function getBrowserInfo() {
+  if (!window || !window.navigator || !window.navigator.userAgent) {
+    throw new Error(
+      'You should use @telnyx/webrtc in a web browser such as Chrome|Firefox|Safari'
+    );
+  }
+
+  if (
+    navigator.userAgent.match(/chrom(e|ium)/gim) &&
+    !navigator.userAgent.match(/OPR\/[0-9]{2}/gi) &&
+    !navigator.userAgent.match(/edg/gim)
+  ) {
+    const info = navigator.userAgent
+      .match(/chrom(e|ium)\/[0-9]+\./gim)[0]
+      .split('/');
+    const name = info[0];
+    const version = parseInt(info[1], 10);
+
+    return {
+      browserInfo: navigator.userAgent,
+      name,
+      version,
+      supportAudio: true,
+      supportVideo: true,
+    };
+  }
+
+  if (
+    navigator.userAgent.match(/firefox/gim) &&
+    !navigator.userAgent.match(/OPR\/[0-9]{2}/gi) &&
+    !navigator.userAgent.match(/edg/gim)
+  ) {
+    const info = navigator.userAgent
+      .match(/firefox\/[0-9]+\./gim)[0]
+      .split('/');
+
+    const name = info[0];
+    const version = parseInt(info[1], 10);
+
+    return {
+      browserInfo: navigator.userAgent,
+      name,
+      version,
+      supportAudio: true,
+      supportVideo: false,
+    };
+  }
+
+  if (
+    navigator.userAgent.match(/safari/gim) &&
+    !navigator.userAgent.match(/OPR\/[0-9]{2}/gi) &&
+    !navigator.userAgent.match(/edg/gim)
+  ) {
+    const name = navigator.userAgent.match(/safari/gim)[0];
+    const fullVersion = navigator.userAgent
+      .match(/version\/[0-9]+\./gim)[0]
+      .split('/');
+    const version = parseInt(fullVersion[1], 10);
+    return {
+      browserInfo: navigator.userAgent,
+      name,
+      version,
+      supportAudio: true,
+      supportVideo: true,
+    };
+  }
+
+  if (
+    navigator.userAgent.match(/edg/gim) &&
+    !navigator.userAgent.match(/OPR\/[0-9]{2}/gi)
+  ) {
+    const info = navigator.userAgent.match(/edg\/[0-9]+\./gim)[0].split('/');
+    const name = info[0];
+    const version = parseInt(info[1], 10);
+
+    return {
+      browserInfo: navigator.userAgent,
+      name,
+      version,
+      supportAudio: true,
+      supportVideo: true,
+    };
+  }
+  throw new Error(
+    'This browser does not support @telnyx/webrtc. \nPlease, use Chrome|Firefox|Safari.'
```

Comment:
Users may receive this error even if they're using `Chrome|Firefox|Safari`, IMO we should omit that part of the message:
```suggestion
    'This browser does not support @telnyx/webrtc.'
```

or show the results of `TelnyxRTC.webRTCSupportedBrowserList()` in the message:
```suggestion
    `This browser does not support @telnyx/webrtc. \nSupported browsers: \n${JSON.stringify(webRTCSupportedBrowserList(), null, 2)}`
```

or just mention the `webRTCSupportedBrowserList` method instead:
```suggestion
    'This browser does not support @telnyx/webrtc. To see browser support list: `TelnyxRTC.webRTCSupportedBrowserList()`'
```